.taskcluster.yml

```diff
@@ -1,40 +1,47 @@
 # The version is always required
-version: 0
-allowPullRequests: public
+version: 1
+policy:
+  pullRequest: public
 # Top level metadata is always required
-metadata:
-  name: "Taskcluster Generic Worker Tests"
-  description: "These tests should ensure that any new commits against generic worker codebase are tested across all supported worker types in gekco."
-  owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
-  source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
+#metadata:
```

Comment:
top-level metadata is not needed in v1, so this can go. The comments can be left tho, like this:

```
##############
# Taskcluster Generic Worker Tests
#
# These tests should ensure that any new commits against generic worker codebase are tested across all supported worker types in gecko.
# 
##############
```